arm-network/2017-06-01/swagger/applicationGateway.json

```diff
@@ -0,0 +1,1912 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2017-06-01"
```

Comment:
Are you only changing the api-version of this file? For all other files, its still 2017-03-01 even though they are moved to the 2017-06-01 folder? It would be difficult to review if you add the all the files to a new folder even though nothing has changed.